Hotfix

ddudt opened the pull request:
- IO formatting changes
- 3D plotting
- utility function reorganization

f0uriest commented:
What is the writer_test_file on  #0656ef5? Is that needed by some of the tests? if so we should put it in with them, and give it a proper file extension (h5, nc, pkl etc).

ddudt commented:
`writer_test_file` is automatically generated during one of the tests.  We need to change the test to put it in a temporary directory so it gets deleted after the test runs.  I added the file to the repo by accident, and have now removed it.

f0uriest reviewed (approved):
OK, looks good.
We can probably clean up the backend/utils a bit more, I'd like to get rid of the TextColors thing in favor of a cross platform alternative like colorama but we can do that later.
